examples/f16/components/fgnetfdm.py

```diff
@@ -64,32 +59,32 @@ class FGNetFDM:
 
     # Engine status
     num_engines: int = 0                       # Number of valid engines
-    eng_state: [int] # Engine state (off, cranking, running)
-    rpm: [int]          # Engine RPM rev/min
-    fuel_flow: [int]    # Fuel flow gallons/hr
-    fuel_px: [int]      # Fuel pressure psi
-    egt: [int]          # Exhuast gas temp deg F
-    cht: [int]          # Cylinder head temp deg F
-    mp_osi: [int]       # Manifold pressure
-    tit: [int]          # Turbine Inlet Temperature
-    oil_temp: [int]     # Oil temp deg F
-    oil_px: [int]       # Oil pressure psi
+    eng_state: typing.List[int] # Engine state (off, cranking, running)
```

Comment:
It might be preferable to use `typing.Tuple` if they don't change over time (immutable) or simply `typing.Sequence` to annotate sliceable data structures.

Reply:
Changed to `typing.Sequence`